lib/javascripts/messageformat-lookup.js

```diff
@@ -1,9 +1,13 @@
 (function () {
   I18n.messageFormat = function (key, options) {
```

Comment:
Is there any reason we should keep this as a separate file?

Feels like we could probably just define the `I18n.messsageFormat` function in `discourse-i18n/src/index.js`?